app/sdk/quests/questFactory.coffee

```diff
@@ -215,7 +215,7 @@ class QuestFactory
 				return 0
 
 			# Player can't use any cards with cost greater than 3 to make progress
-			 playerGameSetupData = UtilsGameSession.getPlayerSetupDataForPlayerId(gameSessionData,playerId)
+			playerGameSetupData = UtilsGameSession.getPlayerSetupDataForPlayerId(gameSessionData,playerId)
```

Comment:
This may or may not be a behavioral change. Had accidental whitespace/indentation previously.